dbdimp.h

```diff
@@ -110,22 +124,22 @@
  */
 
 /* Use mysql_options with MYSQL_OPT_SSL_VERIFY_SERVER_CERT */
-#if ((MYSQL_VERSION_ID >= 50023 && MYSQL_VERSION_ID < 50100) || MYSQL_VERSION_ID >= 50111) && (MYSQL_VERSION_ID < 80000 || defined(MARIADB_BASE_VERSION))
+#ifdef CLIENT_SSL_VERIFY_SERVER_CERT
```

Comment:
Yes, this looks better, I will fix and push later today. We need to think about to add ssl_mode also in Connector/C, so API's will not differ too much and iit would be easier to handle TLS/SSL options in the future (e.g. Connector/C additionally supports finger print check of server certificate).